### DISCUSSION TITLE
FACTURACION ELECTRONICA MEXICO

### DISCUSSION THREAD

**Discussion by barradasmtz1254** [frappe/books#603] [General]
Buenas noches, quisera saber si se puede facturar electroicamente conforme los lineamientos del SAT mexico , para genear CFDI

**Comment by MaxMorais** [comment]
@barradasmtz1254 there's no integration with Electronic Invoices by default in Frappe Books!
but, we i'm interested to bring that integration if needed.

@barradasmtz1254 no hay la integración para la facturación electronica por defecto en Frappe Books.
aun que, me interessa hacer-la. En el passado you ya hice una integracion con facturacion moderna, y no suena complicada hacer-la en frappe books.
